client/tinychain/graph/app.py

```diff
@@ -84,18 +84,16 @@ class Graph(App):
     a table column which has an edge to itself is an `Edge`, otherwise it's a `ForeignKey`. `ForeignKey` relationships
     are automatically updated when a `Table` is updated, but `Edge` relationships require explicit management.
     """
-    schema = None
 
     # TODO: remove the `chain_type` parameter and generate the initial schema via reflection
     def __init__(self, models: list[Model] = None, schema: Schema = None, chain_type=Sync):
```

Comment:
you can remove the `models` argument since you added the `autogenerate` classmethod